dask/dataframe/io/csv.py

```diff
@@ -484,10 +484,13 @@ def read_pandas(
         lineterminator = "\n"
     if include_path_column and isinstance(include_path_column, bool):
         include_path_column = "path"
-    if "index" in kwargs or "index_col" in kwargs:
+    if "index" in kwargs or (
+        "index_col" in kwargs and kwargs.get("index_col") is not False
+    ):
         raise ValueError(
-            "Keywords 'index' and 'index_col' not supported. "
-            f"Use dd.{reader_name}(...).set_index('my-index') instead"
+            "Keywords 'index' and 'index_col' not supported (except for "
+            "'index_col=False' to force pandas to not use the first column as "
```

Comment:
Not sure if the to force pandas... part is needed. This follows from the pandas docs

Reply:
OK, removed that